tests/neg/i16453.check

```diff
@@ -0,0 +1,18 @@
+-- [E172] Type Error: tests/neg/i16453.scala:14:21 ---------------------------------------------------------------------
+14 |  summon[Option[Int]] // error
+   |                     ^
+   |No given instance of type Option[Int] was found for parameter x of method summon in object Predef
+   |
+   |Note: implicit conversions are not automatically applied to arguments of using clauses. You will have to pass the argument explicitly.
+   |The following conversions in scope result in Option[Int]: 
```

Comment:
```suggestion
   |The following conversions in scope might result in Option[Int]: 
```
As this example shows `Option[Int]` is just one of possible outputs of the mentioned conversions, given that `T =:= Int`.

And now, if I understand it correctly, all the conversions potentially resulting in `T` are listed, not only the ones for which there is an implicit value, which could be converted to `T`, right? In this case this additional message is not really helpful IMO. The point of it was that there is some implicit value which was not picked by the implicit search because an implicit conversion was not applied on it at the end.

Actually now I realized I slightly misread your previous comment or my answer was not clear

> Would it be worth it to filter out Conversions for which there aren't any implicits to feed to? For example, in the code below, there's no `given Int` to feed into the conversion. Should it also check for that or just include any `Conversion`s found that result in `Option[Int]`s?
> 
> ```scala
> given Conversion[Int, Option[Int]] = ???
> summon[Option[Int]]
> ```

For some reason I though you by default included only these conversions, for which there is an implicit value to be converted to the expected type and you wanted to include the remaining conversions as an additional feature to give users some more information. But adding these would not be helpful unless you clearly separated these two categories of conversions and gave some more specific explanation about them in the error message.
So for now I would suggest filtering out these conversions, for which the implicit value to convert from is missing. Sorry for the confusion.

Reply:
Got it, I'll try to filter and keep only the conversions where "chaining" could have happened.